### ISSUE TITLE
Add support for EL7

### ISSUE THREAD

**Issue by lorengordon** [plus3it/splunkforwarder-formula#4]
Currently, the formula fails on EL7 due to the `iptables` states:

```
----------
          ID: Allow Splunk Mgmt Outbound Port 8089
    Function: iptables.append
      Result: False
     Comment: State 'iptables.append' was not found in SLS 'splunkforwarder.elx'
              Reason: 'iptables' __virtual__ returned False
     Changes:   
----------
          ID: Allow Splunk Mgmt Outbound Port 9997
    Function: iptables.append
      Result: False
     Comment: State 'iptables.append' was not found in SLS 'splunkforwarder.elx'
              Reason: 'iptables' __virtual__ returned False
     Changes:  
```

**Comment by ferricoxide** [comment]
This will likely be a non-trivial change: firewalld subsumes much of the iptables functionality (just to run iptables in legacy mode, you have to install iptables service and disable firewalld ...which STIGs declare a no-no).

**Comment by lorengordon** [comment]
it's pretty common in an sls to branch based on the system-type to support different mechanisms that achieve the same thing. sometimes that logic is moved into a separate file and imported into the sls to keep the sls more tidy.

```
{%- if grains['osmajorrelease'] in ['7'] %}
# do el7 firewalld stuff
{%- elif grains['osmajorrelease'] in ['6'] %}
# do el6 iptables stuff
{%- endif %}
```

**Comment by ferricoxide** [comment]
Yeah: had to do similar branching for a `yum.conf` handler for EL7: what STIGs want for RHEL cause major problems if you don't prevent it on CentOS.

**Comment by ferricoxide** [comment]
Completely forgot this issue was here: did not encounter issues with this week's testing with RH7 and CO7.

Realized, this afternoon, "the formula's currently only split Windows/Linux but not further branched to EL6/EL7". Came to the issues page for this project to file a "do it the right way" kind of (enhancement) issue. and re-found this issue. Now I need to go back and double-check that my recollection of successes was valid!

**Comment by ferricoxide** [comment]
Closed by PR #5
